tools/copycat/ccsra.c

```diff
@@ -208,12 +225,15 @@ static rc_t CC CCNodeSraDirResolvePath (const CCNodeSraDir *self,
                                         const char *path_fmt,
                                         va_list args)
 {
-    char path[4096];
-    int size = args ?
-        vsnprintf ( path, sizeof path, path_fmt, args ) :
-        snprintf  ( path, sizeof path, "%s", path_fmt );
-    if ( size < 0 || size >= (int) sizeof path )
-        return RC (rcFS, rcNoTarg, rcAccessing, rcPath, rcExcessive );
+    char buffer[4096];
```

Comment:
Does not call `CCNodeSraDirLegalPath` or its new version, need to do the valist handling itself.